tutorial/practicals/nemo/2_nemo_profiling/README.md

```diff
@@ -49,11 +49,11 @@ this session.)
        $ make allclean
        $ make tra_adv.exe
 
-   At this point, the compiled application can be run (ensure you have
+   At this point, the compiled application can be run (either ensure you have
    the necessary environment variables set first - see
-   ../1_nemo_psyir/README.md):
+   ../1_nemo_psyir/README.md - or include them inline as indicated):
```

Comment:
Could you make it a clickable "Tutorial 1" link/

Reply:
Yes, done.